module-services/service-desktop/WorkerDesktop.cpp

```diff
@@ -161,14 +162,18 @@ bool WorkerDesktop::handleMessage(uint32_t queueID)
                                                   sdesktop::usb::USBConfigurationType::reconfiguration),
                                               service::name::service_desktop);
             }
+            usbStatus = bsp::USBDeviceStatus::Configured;
```

Comment:
check if it doesn't break shutdown behaviour :)

Reply:
good point, please also check if CDC is still working on simualtor

Reply:
@pholat Phone turning off when USB connected work flawlessly. 
@alekrudnik  USB for simulator wokrs fine as proven with harness test
![sim_usb](https://user-images.githubusercontent.com/74544942/141136914-0d1e9d6e-13c4-40a4-9a2d-10007ab46a32.gif)
: